cts/cts.in

```diff
@@ -156,15 +156,21 @@ if [ x$cmd != x ]; then
 	    ;;
 	trace-add|tadd)
 	    echo "Adding $trace to PCMK_trace_functions"
-	    cluster-helper -g $cluster_name -- sed -i "s/.*PCMK_trace_functions=/PCMK_trace_functions=$trace,/" $config
+	    cp "$config" "$config.$$"
+	    cluster-helper -g $cluster_name -- sed -e "s/.*PCMK_trace_functions=/PCMK_trace_functions=$trace,/" "$config" > "$config.$$"
+	    mv -- "$config.$$" "$config"
```

Comment:
The changes in this file won't work -- cluster-helper logs into each node and runs the given command. Not sure whether the `\&\&` trick will work with cluster-helper.

I think the point of calling `cp` first is to use `-p` so the new file has the same permissions as the old. Otherwise it's redundant.

Might be worth functionizing if argument quoting isn't too much of a pain

Reply:
Ah right, yeah - I think I realized that about cluster-init.in and then didn't go back and fix cts.in.  I'll change that.  The `\&\&` thing should work.

Reply:
The only problem with functionizing it is I don't think we have any shell library I can source from all these places, do we?  It would have to be accessible from cts/, maint/, and tools/.

Reply:
I wasn't thinking of a library, just duplicating it where needed